### ISSUE TITLE
Outdated tokio-current-thread dependency

### ISSUE THREAD

**Issue by ilammy** [tokio-rs/tokio#668]
I've updated to recently released `tokio` 0.1.9 and encountered the following weird compilation error on Linux:

```
    Checking tokio v0.1.9                                                       
error[E0599]: no method named `status` found for type `tokio_current_thread::Handle` in the current scope
  --> /home/ilammy/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-0.1.9/src/runtime/current_thread/runtime.rs:56:16
   |
56 |         self.0.status()
   |                ^^^^^^

error: aborting due to previous error
```

It seems to be caused by an outdated `tokio-current-thread = "0.1.1"` dependency in `Cargo.toml` of `tokio`. The `tokio-current-thread` crate has been updated to 0.1.2 together with `tokio` 0.1.9 and it seems `tokio` needs this update. However, my Cargo.toml does not specify `tokio-current-thread` dependency explicily and my Cargo.lock file specified 0.1.1 for tokio-current-thread. It did not update automatically. Manually specifying `tokio-current-thread = "0.1.2"` in my Cargo.toml resolves the compilation error.

To reproduce make an empty project with `cargo init` and add the following to Cargo.toml:

```toml
[dependencies]
tokio = "=0.1.9"
tokio-current-thread = "=0.1.1"
```

**Comment by tobz** [comment]
@ilammy Woops!  I'll let @carllerche know and see if we can get this resolved today.

**Comment by tobz** [comment]
This should be all fixed with the tokio 0.1.10 release.

Feel free to open another issue if you see something wrong, and thanks for letting us know about this one! :)


-toby
